### PR TITLE
Landing page for default browser campaign [Fixes: 8431]

### PR DESCRIPTION
**Pull request by Errolyn**
## Description
Build landing page for the default browser campaign

## Issue / Bugzilla link
#8431

### CODE REVIEW

**Comment by Errolyn** [comment]
It's ready for review, I need to add the utm but wasn't sure what should be there.

File: bedrock/firefox/templates/firefox/set-as-default/landing-page.html [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+{% add_lang_files "firefox/whatsnew_73" %}
+
+{% extends "firefox/base/base-protocol.html" %}
+{% from "macros-protocol.html" import hero, picto_card with context %}
+
+{% block page_title_prefix %}{{_('Make Firefox your default browser')}} — {% endblock %}
+{% block page_desc %}{{ _('Choose the browser that protects your privacy. Set Firefox as your default browser.') }}{% endblock %}
+
+{% set utms = '' %}
```
Comment (alexgibson) [inline bedrock/firefox/templates/firefox/set-as-default/landing-page.html]: TODO: utms

File: media/img/firefox/set-as-default/landing/browserdefault_freedom_icon.svg [hunk 2]
```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg width="48px" height="42px" viewBox="0 0 48 42" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+    <!-- Generator: Sketch 59.1 (86144) - https://sketch.com -->
```
Comment (alexgibson) [inline media/img/firefox/set-as-default/landing/browserdefault_freedom_icon.svg]: TODO: optimize svg's

File: bedrock/firefox/templates/firefox/set-as-default/landing-page.html [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% add_lang_files "firefox/whatsnew_73" %}
+
+{% extends "firefox/base/base-protocol.html" %}
+{% from "macros-protocol.html" import hero, picto_card with context %}
+
+{% block page_title_prefix %}{{_('Make Firefox your default browser')}} — {% endblock %}
```
Comment (alexgibson) [inline bedrock/firefox/templates/firefox/set-as-default/landing-page.html]:
```suggestion
{% block page_title %}{{_('Make Firefox your default browser')}}{% endblock %}
```

File: bedrock/firefox/templates/firefox/set-as-default/landing-page.html (hunk 1 above)
Comment (alexgibson) [reply bedrock/firefox/templates/firefox/set-as-default/landing-page.html]: Looks like there are no external links on this landing page, so I think we can drop needing the utms here.

**Comment by alexgibson** [comment]
Successful test run: https://gitlab.com/mozmeao/www-config/pipelines/110706455

**Comment by alexgibson** [comment]
@stephaniehobson this PR lgtm, but one last pair of eyes on the changes I added here would be good. Thanks!

File: bedrock/firefox/templates/firefox/set-as-default/landing-page.html [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+    {% call hero(
+      title=_('Make sure you’re protected, every time you get online'),
+      desc=_('Thanks for using the latest Firefox browser. When you choose Firefox, you support a better web for you and everyone else. Now take the next step to protect yourself.'),
+      class='mzp-c-hero',
+      include_cta=True,
+      heading_level=1,
+    ) %}
+    <div class="cta-container">
+      <div class="mzp-c-button-download-container">
+        <a id="set-as-default-button" href="{{ url('firefox.set-as-default.thanks') }}" class="mzp-c-button mzp-t-product">{{ _('Make Firefox your default browser') }}</a>
```
Comment (alexgibson) [inline bedrock/firefox/templates/firefox/set-as-default/landing-page.html]: This CTA should have some data attributes associated with it

File: bedrock/firefox/templates/firefox/set-as-default/landing-page.html [hunk 5]
```diff
@@ -27,7 +27,9 @@
     ) %}
     <div class="cta-container">
       <div class="mzp-c-button-download-container">
-        <a id="set-as-default-button" href="{{ url('firefox.set-as-default.thanks') }}" class="mzp-c-button mzp-t-product">{{ _('Make Firefox your default browser') }}</a>
+        <a id="set-as-default-button" href="{{ url('firefox.set-as-default.thanks') }}" class="mzp-c-button mzp-t-product" data-link-name="Make Firefox your default browser" data-link-type="button">
```
Comment (stephaniehobson) [inline bedrock/firefox/templates/firefox/set-as-default/landing-page.html]: I think we're moving to `data-cta-text` and `data-cta-type` for analytics now.

**Comment by alexgibson** [comment]
Thanks @stephaniehobson updated. I also fixed the analytics data attributes on the WNP, and the icons on the default/thanks/ page here as well. Nice catch!

**Comment by stephaniehobson** [comment]
Thanks! R+ 🔭 

Merge at will.

**Pull request merged**